contracts/external/traders/StopLimitOrders.sol

```diff
@@ -502,6 +536,37 @@ contract LimitOrders is
         );
     }
 
+    /**
+     * Verifies that the order is decreasing the size of the maker's position.
+     */
+    function verifyDecreaseOnly(
+        Types.Par memory oldInputPar,
+        Types.Par memory newInputPar,
+        Types.AssetAmount memory assetAmount,
+        Account.Info memory makerAccount,
+        uint256 outputMarketId
+    )
+        private
+        view
+    {
+        // verify inputMarketId
+        Require.that(
+            newInputPar.isZero()
+            || (newInputPar.value <= oldInputPar.value && newInputPar.sign == oldInputPar.sign),
+            FILE,
+            "inputMarket not decreased"
+        );
+
+        // verify outputMarketId
+        Types.Wei memory oldOutputWei = SOLO_MARGIN.getAccountWei(makerAccount, outputMarketId);
+        Require.that(
+            assetAmount.value == 0
+            || (assetAmount.value <= oldOutputWei.value && assetAmount.sign != oldOutputWei.sign),
```

Comment:
how do you know the `assetAmount` is in `wei`?

Reply:
It always is (hardcoded)

Reply:
As in Solo hardcodes it? Do you mind pointing me to that?

Reply:
https://github.com/dydxprotocol/solo/pull/379/files#diff-4c3acf038be80f22890411b750d3ead8R605 hardcoded in StopLimitOrders.sol